### ISSUE TITLE
PSR2.Classes.PropertyDeclaration.Multiple throws false positives when value is a variable

### ISSUE THREAD

**Issue by tvdijen** [PHPCSStandards/PHP_CodeSniffer#257]
## Describe the bug
PSR2.Classes.PropertyDeclaration.Multiple throws false positives when the default value is a variable.

### Code sample
```php
use Some\Other\Class as C;

class A
{
    public static array $SUPPORTED_SIGNATURE_ALGORITHMS = C::$RSA_DIGESTS;
}
```

PHPCS output here

## Expected behavior
I expect the test to pass. Instead the following error is displayed:
There must not be more than one property declared per statement (PSR2.Classes.PropertyDeclaration.Multiple)

## Versions (please complete the following information)

| | |
|-|-|
| Operating System | [CentOS 9]
| PHP version | [8.1]
| PHP_CodeSniffer version | [3.8.1]
| Standard | [PSR12]
| Install type | [Composer]

## Please confirm:

- [x] I have searched the issue list and am not opening a duplicate issue.
- [x] I confirm that this bug is a bug in PHP_CodeSniffer and not in one of the external standards.
- [x] I have verified the issue still exists in the `master` branch of PHP_CodeSniffer.

**Comment by jrfnl** [comment]
@tvdijen Eh... that's because that is not valid PHP.... ? https://3v4l.org/KeO3Z

Sorry, but sniffs should not have to deal with fantasy code which isn't supported by PHP itself.

**Comment by tvdijen** [comment]
You are right, I don't know what I was thinking.
